litrpc/listener.go

```diff
@@ -42,12 +43,40 @@ func serveWS(ws *websocket.Conn) {
 	jsonrpc.ServeConn(ws)
 }
 
+// OiOoReadWriter is for One-In-One-Out Reader/Writer and I hope to god that it works.
```

Comment:
very pedantic and nitpicking, but we could do better comments :)

Reply:
It's because as far as I know Go doesn't have a thing that has a way to say "when you call `Read`, call it on this, and when you call `Write` call it on *that*".